docs/Glossary.md

```diff
@@ -13,6 +13,16 @@ description: >
 
 The following terms are arranged in the order of their appearance in the actual user workflow.
 
+### Projects
+**A project is a set of related [Data Scopes](#data-scope), the Data Scopes may contains different [Data Entities](#data-entities) from arbitrary [Data Sources](#data-sources) for Metric Computation**
+
+Project was introduced in v0.15 to solve some complex metric computation problems. For example, There was a **Change Failure Rate** in [DORA Metric](../UserManuals/DORA) which requires connecting Incident(issue.type = 'INCIDENT') to its related Deployment(cicd_task.type = 'DEPLOYMENT') to determine whether a Deployment failed or not. Unfortunately, most data sources do not offer this relationship, the only reasonable approach so far is to connect Incident to its Closest Deployment, without Project, 
+
+A project consists of the following elements:
+A set of Metrics(currently, only DORA)
+A [Blueprint](#blueprints) for active data collection
```

Comment:
1. A [`Blueprint`](#blueprints) to pull data from [supported data sources](https://devlake.apache.org/docs/next/SupportedDataSources). You can change it to the relative path.
2. A set of `Incoming Webhooks` to receive pushed from non-supported data sources